### DISCUSSION TITLE
Why is this not compatible with x64 Macs

### DISCUSSION THREAD

**Discussion by naikrovek** [cirruslabs/tart#189] [General]
I come from a Windows career, so sorry for the potentially stupid question.

Why do the Apple APIs not allow this to work on x64 machines?  Seems like the APIs would work for any supported OS.  Is that not how Apple works, or is there something specific to these APIs that preclude supporting x64?

**Comment by fkorotkov** [comment]
See https://github.com/cirruslabs/tart/issues/67 for more details but TLDR is:

> Unfortunately, we are bounded by what Apple's [Virtualization.Framework](https://developer.apple.com/documentation/virtualization) is providing to us. We might add Intel support along the side of Linux support (https://github.com/cirruslabs/tart/issues/62) but I don't believe Virtualization.Framework will ever support Intel for running macOS. But I don't think we'll ever be able to run both platforms on one machine. 😪

**Reply by naikrovek** [reply]
Ok, thank you.  Apple just doesn't support it with the API you're using.  That's surprising to me, but that's how it is.

AArch64 Linux would be useful, but not so useful that I would ask for it.  I have a storage bin full of arm64 development boards that run Linux, so I have no need.

Thanks.
